scripts/tests/gaia_tests.bats

```diff
@@ -58,29 +58,15 @@ setup() {
 ##############################################################################################
 
 @test "[INTEGRATION-BASIC] address names are correct" {
-  assert_equal $STRIDE_ADDRESS "stride1uk4ze0x4nvh4fk0xm4jdud58eqn4yxhrt52vv7"
-  assert_equal $GAIA_ADDRESS "cosmos1pcag0cj4ttxg8l7pcg0q4ksuglswuuedcextl2"
-
-  assert_equal $DELEGATION_ICA_ADDR "cosmos1sy63lffevueudvvlvh2lf6s387xh9xq72n3fsy6n2gr5hm6u2szs2v0ujm"
-  assert_equal $REDEMPTION_ICA_ADDR "cosmos1xmcwu75s8v7s54k79390wc5gwtgkeqhvzegpj0nm2tdwacv47tmqg9ut30"
-  assert_equal $WITHDRAWAL_ICA_ADDR "cosmos1x5p8er7e2ne8l54tx33l560l8djuyapny55pksctuguzdc00dj7saqcw2l"
-  assert_equal $REVENUE_EOA_ADDR "cosmos1wdplq6qjh2xruc7qqagma9ya665q6qhcwju3ng"
-  assert_equal $FEE_ICA_ADDR "cosmos1lkgt5sfshn9shm7hd7chtytkq4mvwvswgmyl0hkacd4rmusu9wwq60cezx"
-  assert_equal $GAIA_DELEGATE_VAL_1 "cosmosvaloper1pcag0cj4ttxg8l7pcg0q4ksuglswuuedadj7ne"
-  assert_equal $GAIA_DELEGATE_VAL_2 "cosmosvaloper133lfs9gcpxqj6er3kx605e3v9lqp2pg5syhvsz"
+  assert_equal $(STRIDE_ADDRESS) "stride1uk4ze0x4nvh4fk0xm4jdud58eqn4yxhrt52vv7"
+  assert_equal $(GAIA_ADDRESS) "cosmos1pcag0cj4ttxg8l7pcg0q4ksuglswuuedcextl2"
+
+  assert_equal $GAIA_DELEGATION_ICA_ADDR "cosmos1sy63lffevueudvvlvh2lf6s387xh9xq72n3fsy6n2gr5hm6u2szs2v0ujm"
+  assert_equal $GAIA_REDEMPTION_ICA_ADDR "cosmos1xmcwu75s8v7s54k79390wc5gwtgkeqhvzegpj0nm2tdwacv47tmqg9ut30"
+  assert_equal $GAIA_WITHDRAWAL_ICA_ADDR "cosmos1x5p8er7e2ne8l54tx33l560l8djuyapny55pksctuguzdc00dj7saqcw2l"
+  assert_equal $GAIA_FEE_ICA_ADDR "cosmos1lkgt5sfshn9shm7hd7chtytkq4mvwvswgmyl0hkacd4rmusu9wwq60cezx"
+  assert_equal $GAIA_DELEGATE_VAL "cosmosvaloper1pcag0cj4ttxg8l7pcg0q4ksuglswuuedadj7ne"
   assert_equal $GAIA_RECEIVER_ACCT "cosmos1g6qdx6kdhpf000afvvpte7hp0vnpzapuyxp8uf"
```

Comment:
Thanks for cleaning this up. 

Noticing that the int tests across chains are sufficiently similar that I think we could probably write a single well-paramaterized `chain_int_tests.bats` and pass in the chain names we'd like to test in the same way as we do for build. For example, we could call `make test-integration chains=gsj` to test gaia stargaze and juno, but not osmosis. 

Don't think this is worth doing now. Maybe after the next chain?

Reply:
Yup I was thinking the same thing! Also I got a slight head start on this if you look at the [stars integration tests](https://github.com/Stride-Labs/stride/blob/stars-dockernet/scripts/tests/stars_tests.bats#L63-L67) where the validator and ICA addresses are generated dynamically. 

I didn't want to go through the headache of updating GAIA,OSMO,JUNO atm since it would have probably warranted a couple more hours of running tests haha but planning to do it when I add the next host. 

And it's still a little unclear to me how involved the full refactor would be (i.e. one single parameterized script) - but I'll re-evaluate as I add the next host!